src/Elcodi/Plugin/FacebookBundle/Resources/translations/messages.es.yml

```diff
@@ -0,0 +1,15 @@
+facebook_plugin:
+    title: Facebook
+    field:
+        facebook_account:
+            title: Cuenta de facebook
```

Comment:
In this translation file you mix `facebook` and `Facebook` indistinctively. In the original `en` translation file, you always use `Facebook`. I think that's a good idea to be more consistent.